src/sage/combinat/root_system/type_D.py

```diff
@@ -117,7 +117,9 @@ def fundamental_weight(self, i):
 
 
 from sage.misc.persist import register_unpickle_override
-register_unpickle_override('sage.combinat.root_system.type_A', 'ambient_space',  AmbientSpace)
+register_unpickle_override('sage.combinat.root_system.type_A',
+                           'ambient_space',
+                           AmbientSpace)
```

Comment:
Perhaps I failed to mention this too.

Reply:
?

Reply:
indeed, sorry to forget about this duplication. In fact those lines are quite wrong. This is a file about type D, nothing about type A there. So I removed them.

Reply:
OK. Thanks.